docs/proposals/receive_split.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Thanos Remote Write
```

Comment:
Is this relevant change?

Reply:
This is not my change this is attached to my PR due to the wrong rebase, I have fixed it in the new PR #3552 .